releasenotes/notes/add-runtime-and-sampling-rules-to-startup-logs-18a8a8236d4131f5.yaml

```
@@ -0,0 +1,4 @@
+---
+upgrade:
```

```suggestion
features:
```


or maybe

```suggestion
other:
```

"upgrade" is like "in order to upgrade you should make X change"